libs/operation_mode/inc/public/tfc/operation_mode/common.hpp

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include <cstdint>
+#include <string_view>
+
+#include <magic_enum.hpp>
+
+#include <tfc/dbus/string_maker.hpp>
+#include <tfc/stx/basic_fixed_string.hpp>
+#include <tfc/stx/string_view_join.hpp>
+#include <tfc/stx/to_tuple.hpp>
+
+namespace tfc::operation {
+
+enum struct mode_e : std::uint8_t {
+  unknown = 0,
+  stopped = 1,
+  running = 2,
+  specialized_running_1 = 3,
+  specialized_running_2 = 4,
+  specialized_running_3 = 5,
```

Comment:
Remove specialized for now, lets revisit if the need arrises. Add starting and stopping modes for initalization and shutdown phases of machine runtime cycle.

Reply:
removed in next PR